dev/com.ibm.ws.webcontainer.security.app/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -91,8 +91,7 @@ trackLoggedOutSSOCookies=Track logged out LTPA single signon tokens
 trackLoggedOutSSOCookies.desc=Specifies whether to track LTPA single signon tokens that are logged out on a server so that it can not be reused on the same server.
 
 overrideHttpAuthenticationMechanism=Name of HttpAuthenticationMechanism to override
-overrideHttpAuthenticationMechanism.desc=Specifies the HttpAuthenticationMechanism for overriding an application-defined HttpAuthenticationMechanism. The Java EE 8 Security specification defines the HttpAuthenticationMechanism. The acceptable value is BASIC or FORM. When FORM is used, specify the loginFormURL and loginErrorURL properties.
-
+overrideHttpAuthenticationMechanism.desc=Specifies the login configuration for overriding an application defined login configuration. The acceptable value is BASIC, FORM or CLIENT_CERT. When BASIC or FORM is used, it is specifies the HttpAuthenticationMechanism, of which the JAVA EE 8 security specification defines, for overriding an application-defined HttpAuthenticationMechanism. When CLIENT_CERT is used, the client certificate authentication takes precedence for the original login configuration of all applications.
```

Comment:
Change: When BASIC or FORM is used, it is specifies the HttpAuthenticationMechanism, of which the JAVA EE 8 security specification defines, for overriding an application-defined HttpAuthenticationMechanism. When CLIENT_CERT is used, the client certificate authentication takes precedence for the original login configuration of all applications.
To:
The BASIC or FORM value specifies the HttpAuthenticationMechanism bean for overriding an application-defined HttpAuthenticationMechanism bean. When you use the CLIENT_CERT value, the client certificate authentication takes precedence for the original login configuration of all applications.

Reply:
Thank you for your review, unfortunately, the message has been revised after your review. Would you mind reviewing the latest one again?